### PR TITLE
add import notification

### PR DESCRIPTION
**Pull request by peizhou298**
## Description
Describe the changes in this PR.

## Related issues
Addresses [issue #].

## Testing
Describe how this change was tested.

## FHIR Team Checklist
- [ ] **Update the title** of the PR to be succinct and less than 50 characters
- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [ ] Tag the PR with Azure API for FHIR if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
- [ ] Tag the PR with Azure Healthcare APIs if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
- [ ] CI is green before merge
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportOrchestratorTask.cs [hunk 1]
```diff
@@ -310,8 +343,12 @@ private async Task UpdateProgressAsync(ImportOrchestratorTaskContext context, Ca
                 result[input.Url] = processingTask;
 
                 beginSequenceId = endSequenceId;
+
+                totalSizeInBYtes += blobSizeInBytes;
```
Comment (tongwu-msft) [inline src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportOrchestratorTask.cs:347]: Typo

File: src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportTaskMetricsNotification.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+        public string Id { get; }
+
+        public string Status { get; }
+
+        public DateTimeOffset CreatedTime { get; }
+
+        public DateTimeOffset EndTime { get; }
+
+        public long? DataSize { get; }
```
Comment (tongwu-msft) [inline src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportTaskMetricsNotification.cs:43]: why ? here?

File: src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Import/ImportOrchestratorTaskTests.cs [hunk 3]
```diff
@@ -62,6 +63,7 @@ public async Task GivenAnOrchestratorTaskAndWrongEtag_WhenOrchestratorTaskStart_
             ImportOrchestratorTaskInputData importOrchestratorTaskInputData = new ImportOrchestratorTaskInputData();
             ImportOrchestratorTaskContext importOrchestratorTaskContext = new ImportOrchestratorTaskContext();
             ITaskManager taskManager = Substitute.For<ITaskManager>();
+            IMediator mediator = Substitute.For<IMediator>();
```
Comment (tongwu-msft) [inline src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Import/ImportOrchestratorTaskTests.cs:66]: we can check here for unit test.

File: src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportOrchestratorTask.cs (hunk 1 above)
Comment (peizhou298) [reply src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportOrchestratorTask.cs]: fixed

**Pull request closed without merging**